metecho/api/filters.py

```diff
@@ -23,7 +24,7 @@ class GitHubIssueFilter(filters.FilterSet):
 
     class Meta:
         model = GitHubIssue
-        fields = ("project",)
+        fields = ("project", "id")
```

Comment:
@jerivas I know it's a bit odd to filter the list by `id` rather than fetch a resource directly, but this allowed us to reuse existing front-end logic. Is this okay with you?

Reply:
```suggestion
        fields = ("project", "id")  # Filtering by ID let's the frontend reuse existing logic
```
That's ok with me, would just add a comment for clarity